provisioning_virtual_machines_and_hosts/_topics/fulfilling_requests.md

```diff
@@ -22,6 +22,7 @@ information on state machines, see [State Machines](#state-machines).
 | Register DNS        | Integrates with IPAM to register with DNS.                                          |
 | Register CMDB       | Integrates with CMDB (Configuration Management Database) to register with the CMDB. |
 | Register AD         | Integrates with IPAM to register with active directory.                             |
+| Placement           | Details of a state instance
```

Comment:
This description is wrong.  I think it should be something like `Determine placement of virtual machine or instance`

Reply:
Thanks, I've made changes.